automates/text_reading/src/main/scala/org/clulab/aske/automates/OdinEngine.scala

```diff
@@ -81,13 +81,30 @@ class OdinEngine(
 
     // Run the main extraction engine, pre-populated with the initial state
     val events =  engine.extractFrom(doc, initialState).toVector
-    //println(s"In extractFrom() -- res : ${res.map(m => m.text).mkString(",\t")}")
-    val (descriptionMentions, other) = events.partition(_.label.contains("Description"))
 
-    val untangled = loadableAttributes.actions.untangleConj(descriptionMentions)
+    // process context attachments to the initially extracted mentions
+    val newEventsWithContexts = loadableAttributes.actions.makeNewMensWithContexts(events)
+    val (contextEvents, nonContexts) = newEventsWithContexts.partition(_.label.contains("ContextEvent"))
+    val mensWithContextAttachment = loadableAttributes.actions.processRuleBasedContextEvent(contextEvents)
+
+    for (m <- mensWithContextAttachment) {
```

Comment:
I forgot to delete this printout, would you please? (All the printout loops here, at least the ones starting on line 90, 94, and 100 from what I see)

Reply:
No worries, they are gone now!